src/core/src/main/java/org/apache/jmeter/gui/action/HtmlReportGenerator.java

```diff
@@ -83,7 +84,7 @@ public List<String> run() {
             LOGGER.debug("Running report generation");
             resultCode = sc.run(generationCommand);
             if (resultCode != 0) {
-                errorMessageList.add(commandExecutionOutput.toString());
+                errorMessageList.add(commandExecutionOutput.toString(StandardCharsets.UTF_8.name()));
```

Comment:
Does this really work? Is there any configuration that enforces that `SystemCommand` outputs the data in UTF-8?

Reply:
Can't really say. What I can say is there is a not small amount of code in here that assumes `UTF8`. Or should this fall back to `Charset.defaultCharset()` instead?

Reply:
Replacing `.toString()` with `.toString(...defaultCharset...)` is a step forward to identifying the locations that rely on default charset. However, it would be great if we can move towards UTF-8 since UTF-8 will be the default in the upcoming Java.

Reply:
Got it. How about the ones in `HttpRequestHdr`? They seem to be unambiguously `ISO_8859_1`.

Reply:
They are fine with ISO